utilities/general.py

```diff
@@ -171,3 +179,75 @@ def create_isvc_label_selector_str(isvc: InferenceService, resource_type: str, r
 
     else:
         raise ValueError(f"Unknown deployment mode {deployment_mode}")
+
+
+def get_csv_related_images(admin_client: DynamicClient, csv_name: str | None = None) -> List[Dict[str, str]]:
+    """Get relatedImages from the CSV.
+
+    Args:
+        admin_client: The kubernetes client
+        csv_name: Optional CSV name. If not provided, will use {operator_name}.{version}
+                 where operator_name is determined by the distribution (rhods-operator for OpenShift AI,
+                 opendatahub-operator for Open Data Hub)
+
+    Returns:
+        List of related images from the CSV
+    """
+    from utilities.infra import (
```

Comment:
You can import utilities.infra to avoid circular import

Reply:
I can't, because infra imports utilities.general